apps/playground-web/src/components/engine/airdrop/TransactionResults.tsx

```diff
@@ -0,0 +1,278 @@
+          Transaction Results
+        </h2>
+        <span className="text-muted-foreground text-sm">
+          Last 24 hours • {sortedResultsWithDummy.length} transactions
+        </span>
+      </CardHeader>
+      <CardContent className="relative max-h-[400px]">
+        <div className="overflow-x-auto" onScroll={handleScroll}>
+          <Table>
+            <TableHeader>
+              <TableRow>
+                <TableHead className="min-w-[130px]">Queue ID</TableHead>
+                <TableHead className="min-w-[120px]">Network</TableHead>
+                <TableHead className="min-w-[130px]">From</TableHead>
+                <TableHead className="min-w-[130px]">Queued</TableHead>
+                <TableHead className="min-w-[100px]">Status</TableHead>
+                <TableHead className="min-w-[160px]">Tx Hash</TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              {currentItems.map((result, index) => (
+                <TableRow key={`${result.network}-${result.queueId}-${index}`}>
+                  <TableCell className="font-medium">
+                    {result.queueId
+                      ? `${result.queueId.substring(0, 6)}...${result.queueId.substring(
+                          result.queueId.length - 4,
+                        )}`
+                      : "----"}
+                  </TableCell>
+                  <TableCell>
+                    <span className="flex items-center gap-2">
+                      {result.network === "Base Sep" && (
+                        <img
+                          src="/BaseSep.png"
+                          alt="Base"
+                          className="h-4 w-4"
+                        />
+                      )}
+                      {result.network === "OP Sep" && (
+                        <img
+                          src="/OP.png"
+                          alt="Optimism Sep"
+                          className="h-4 w-4"
+                        />
+                      )}
+                      {result.network === "Ethereum" && (
+                        <img
+                          src="/Ethereum.png"
+                          alt="Ethereum"
+                          className="h-4 w-4"
+                        />
+                      )}
+                      {result.network}
+                    </span>
+                  </TableCell>
+                  <TableCell>
+                    {(() => {
+                      if (!result.toAddress) return "----";
+
+                      const addressDisplay = `${result.toAddress.substring(0, 6)}...${result.toAddress.substring(
+                        result.toAddress.length - 4,
+                      )}`;
+                      // Keeping OP here for consistency. Will be adding a component for Optimism soon.
+                      const getExplorerUrl = () => {
+                        switch (result.network) {
+                          case "Base Sep":
```

Comment:
Nit: probably enough room to spell out "Sepolia"

Reply:
Fixed, good call out